docs/src/references.md

```diff
@@ -0,0 +1,4 @@
+# References
```

Comment:
Likewise in index.md, the sentence "An illustrative example of AdvancedHMC's usage is given below." is not up-to-date, there's no example below.